docs/en/05_Contributing/03_Documentation.md

```diff
@@ -119,8 +101,7 @@ Code:
 	</div>
 
 <div class="warning" markdown='1'>
-"Warning box": Highlight a severe bug or technical issue requiring a users attention. For example, a code block with 
-destructive functionality might not have its URL actions secured to keep the code shorter.
+"Warning box": A warning box is usefull for highlighting a severe bug or technical issue requiring a user's attention. For example, a code block with destructive functionality might not have its URL actions secured to keep the code shorter (Can a clearer example be given ??).
```

Comment:
Hmm, that example is a bit crappy, do we even need an example of when to use a warning box?!